zk/stages/stage_sequence_execute_utils.go

```diff
@@ -421,3 +422,45 @@ func checkForBadBatch(
 
 	return false, nil
 }
+
+var (
+	LIMIT_128_KB = uint64(128 * 1024)
```

Comment:
we can keep this as a constant 128_000.  This is the limit in the smart contract for batchL2Data rather than "real" kb